src/content/resources/javascript/es6/arrow-functions.md

```diff
@@ -40,12 +41,27 @@ const sayHello = name => {
 };
 ```
 
-Another interesting feature that also reduces the syntax within arrow functions, is that if a function only returns an expression using one line of code, you can omit the `return` keyword and the curly braces. In arrow functions without braces, the `return` is implicit, meaning you don't need to include the `return` keyword. The following function returns "hello":
+Another interesting feature that also reduces the syntax within arrow functions, is that if a function only returns an expression without any 'business logic', you can omit the `return` keyword and the curly braces. In arrow functions without braces, the `return` is implicit, meaning you don't need to include the `return` keyword. The following function returns "hello":
```

Comment:
I would recommend saying 

`if a function is an expression`; this is what that is referred to in other languages, and that's when an arrow function can omit a return statement. Business logic can mean something else often.

Reply:
Alright, will do

Reply:
Perhaps `if a function is an expression with no logic` would be more fitting?

Reply:
A single expression function can still have logic in it, just think about React hooks for example.

Reply:
I suppose